config.py

```diff
@@ -12,3 +12,5 @@ class Config(object):
     SERVICE_URL = os.environ.get("SERVICE_URL")
     SESSION_COOKIE_HTTPONLY = True
     SESSION_COOKIE_SECURE = True
+    BACKEND_API_ENDPOINT = os.environ.get("BACKEND_API_ENDPOINT")
```

Comment:
Don't think the endpoint should be in config, only the hostname.

Reply:
Changed so that only the hostname is specified.